adodbapi/process_connect_string.py

```diff
@@ -1,6 +1,6 @@
 """ a clumsy attempt at a macro language to let the programmer execute code on the server (ex: determine 64bit)"""
 
-from . import is64bit as is64bit
+from . import is64bit

```

Comment:
@vernondcole I try to keep adodbapi changes in separate PRs, but this one had only a single line that has no effect on runtime. This is purely semantics for static tools and checkers (so a user will be told to prefer `from adodbapi import is64bit` instead of `from adodbapi.process_connect_string import is64bit`)